frontend/e2e-tests/page-objects/data_entry/DataEntryHomePgObj.ts

```diff
@@ -34,11 +37,15 @@ export class DataEntryHomePage {
       name: "Je invoer is opgeslagen",
     });
     this.resumeDataEntry = page.getByRole("heading", { level: 2, name: "Je hebt nog een openstaande invoer" });
+    this.dataEntryWarningAlertTitle = page.getByTestId("dataEntryWarningAlertTitle");
```

Comment:
`name` can also take Regex, so we can do:
```suggestion
    this.dataEntryWarningAlertTitle = page.getByRole("heading", {level: 2, name: /^Je kan stembureau \d+ niet invoeren.$/})
```

Reply:
Changed in 1112b3ea1e859892cb9bd5dd0b722701b3258288